CMakeLists.txt

```diff
@@ -177,17 +179,34 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
   set(CMAKE_INSTALL_PREFIX "${CMAKE_SOURCE_DIR}/_dist-${CMAKE_BUILD_TYPE}")
 endif(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 
+# Shared library should be position independent
+set_property(TARGET hiopOptimization PROPERTY POSITION_INDEPENDENT_CODE 1)
+set_property(TARGET hiopLinAlg PROPERTY POSITION_INDEPENDENT_CODE 1)
+set_property(TARGET hiopUtils PROPERTY POSITION_INDEPENDENT_CODE 1)
+set_property(TARGET chiopInterface PROPERTY POSITION_INDEPENDENT_CODE 1)
```

Comment:
You may `set(CMAKE_POSITION_INDEPENDENT_CODE ON)` to enable -fPIC project-wide

Reply:
Fixed 9684cfcf569 .